java-server-side-ui/.gitignore

```diff
@@ -0,0 +1,37 @@
+HELP.md
+.gradle
+build/
+!gradle/wrapper/gradle-wrapper.jar
+!**/src/main/**/build/
+!**/src/test/**/build/
+
```

Comment:
We need to remove the `.mvn/wrapper/maven-wrapper.jar` and add these lines for Maven:
https://github.com/vmware-tanzu/application-accelerator-samples/blob/main/java-rest-service/.gitignore#L7-L10

Reply:
Done